From 78fb29515d869c83dda54b1bd719339838652be2 Mon Sep 17 00:00:00 2001
From: Lior Amsalem <alior@marvell.com>
Date: Sun, 5 Feb 2017 11:55:28 +0200
Subject: [PATCH 1390/2241] configs: net-bm, net-bm-nat defconfig modifications

new feature in CI allows getting the needed defconfig modifications
from a file, this patch adds the relevant configs for net-bm and
net-bm-nat.

Change-Id: I517ec95c14aa421357fbca12250916c15efa602d
Signed-off-by: Lior Amsalem <alior@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/36398
Reviewed-by: Stefan Chulski <stefanc@marvell.com>
Reviewed-by: Hanna Hawa <hannah@marvell.com>
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Omri Itach <omrii@marvell.com>
---
 tools/configs/net-bm     | 39 +++++++++++++++++++++++++++++++
 tools/configs/net-bm-nat | 61 ++++++++++++++++++++++++++++++++++++++++++++++++
 2 files changed, 100 insertions(+)
 create mode 100644 tools/configs/net-bm
 create mode 100644 tools/configs/net-bm-nat

diff --git a/tools/configs/net-bm b/tools/configs/net-bm
new file mode 100644
index 0000000..894e5b6
--- /dev/null
+++ b/tools/configs/net-bm
@@ -0,0 +1,39 @@
+CONFIG_PREEMPT_RCU=n
+CONFIG_KALLSYMS_ALL=n
+CONFIG_SLUB_DEBUG=n
+CONFIG_SLABINFO=n
+CONFIG_UNINLINE_SPIN_UNLOCK=n
+CONFIG_PREEMPT=n
+CONFIG_PREEMPT_COUNT=n
+CONFIG_STAGING=n
+CONFIG_DEBUG_INFO=n
+CONFIG_DEBUG_KERNEL=n
+CONFIG_LOCKUP_DETECTOR=n
+CONFIG_ARM64_PTDUMP=n
+CONFIG_TREE_RCU=y
+CONFIG_INLINE_SPIN_UNLOCK_IRQ=y
+CONFIG_INLINE_READ_UNLOCK=y
+CONFIG_INLINE_READ_UNLOCK_IRQ=y
+CONFIG_INLINE_WRITE_UNLOCK=y
+CONFIG_INLINE_WRITE_UNLOCK_IRQ=y
+CONFIG_CORESIGHT=n
+CONFIG_PREEMPT_NOTIFIERS=n
+CONFIG_ARM64_ERRATUM_834220=n
+CONFIG_MMU_NOTIFIER=n
+CONFIG_VHOST_NET=n
+CONFIG_VHOST_RING=n
+CONFIG_VHOST=n
+CONFIG_HAVE_KVM_IRQFD=n
+CONFIG_HAVE_KVM_EVENTFD=n
+CONFIG_KVM_MMIO=n
+CONFIG_HAVE_KVM_CPU_RELAX_INTERCEPT=n
+CONFIG_KVM_VFIO=n
+CONFIG_HAVE_KVM_ARCH_TLB_FLUSH_ALL=n
+CONFIG_KVM_GENERIC_DIRTYLOG_READ_PROTECT=n
+CONFIG_KVM_COMPAT=n
+CONFIG_VIRTUALIZATION=n
+CONFIG_KVM_ARM_VGIC_V3=n
+CONFIG_KVM=n
+CONFIG_KVM_ARM_HOST=n
+CONFIG_MAGIC_SYSRQ=n
+CONFIG_MEMTEST=n
diff --git a/tools/configs/net-bm-nat b/tools/configs/net-bm-nat
new file mode 100644
index 0000000..e23fca1
--- /dev/null
+++ b/tools/configs/net-bm-nat
@@ -0,0 +1,61 @@
+CONFIG_PREEMPT_RCU=n
+CONFIG_KALLSYMS_ALL=n
+CONFIG_SLUB_DEBUG=n
+CONFIG_SLABINFO=n
+CONFIG_UNINLINE_SPIN_UNLOCK=n
+CONFIG_PREEMPT=n
+CONFIG_PREEMPT_COUNT=n
+CONFIG_STAGING=n
+CONFIG_DEBUG_INFO=n
+CONFIG_DEBUG_KERNEL=n
+CONFIG_LOCKUP_DETECTOR=n
+CONFIG_ARM64_PTDUMP=n
+CONFIG_TREE_RCU=y
+CONFIG_INLINE_SPIN_UNLOCK_IRQ=y
+CONFIG_INLINE_READ_UNLOCK=y
+CONFIG_INLINE_READ_UNLOCK_IRQ=y
+CONFIG_INLINE_WRITE_UNLOCK=y
+CONFIG_INLINE_WRITE_UNLOCK_IRQ=y
+CONFIG_NET_INGRESS=y
+CONFIG_NETFILTER=y
+CONFIG_NETFILTER_ADVANCED=y
+CONFIG_NETFILTER_INGRESS=y
+CONFIG_NETFILTER_NETLINK=y
+CONFIG_NF_CONNTRACK=y
+CONFIG_NF_CONNTRACK_PROCFS=y
+CONFIG_NF_NAT=y
+CONFIG_NF_NAT_NEEDED=y
+CONFIG_NF_TABLES=y
+CONFIG_NETFILTER_XTABLES=y
+CONFIG_NETFILTER_XT_NAT=y
+CONFIG_NF_DEFRAG_IPV4=y
+CONFIG_NF_CONNTRACK_IPV4=y
+CONFIG_NF_CONNTRACK_PROC_COMPAT=y
+CONFIG_NF_NAT_IPV4=y
+CONFIG_NF_NAT_MASQUERADE_IPV4=y
+CONFIG_IP_NF_IPTABLES=y
+CONFIG_IP_NF_FILTER=y
+CONFIG_IP_NF_NAT=y
+CONFIG_IP_NF_TARGET_MASQUERADE=y
+CONFIG_CORESIGHT=n
+CONFIG_PREEMPT_NOTIFIERS=n
+CONFIG_ARM64_ERRATUM_834220=n
+CONFIG_MMU_NOTIFIER=n
+CONFIG_VHOST_NET=n
+CONFIG_VHOST_RING=n
+CONFIG_VHOST=n
+CONFIG_HAVE_KVM_IRQFD=n
+CONFIG_HAVE_KVM_EVENTFD=n
+CONFIG_KVM_MMIO=n
+CONFIG_HAVE_KVM_CPU_RELAX_INTERCEPT=n
+CONFIG_KVM_VFIO=n
+CONFIG_HAVE_KVM_ARCH_TLB_FLUSH_ALL=n
+CONFIG_KVM_GENERIC_DIRTYLOG_READ_PROTECT=n
+CONFIG_KVM_COMPAT=n
+CONFIG_VIRTUALIZATION=n
+CONFIG_KVM_ARM_VGIC_V3=n
+CONFIG_KVM=n
+CONFIG_KVM_ARM_HOST=n
+CONFIG_MAGIC_SYSRQ=n
+CONFIG_MEMTEST=n
+
-- 
2.7.4

